### ISSUE TITLE
@beamwind/ssr doesn't exist -> readme SSR example is broken

### ISSUE THREAD

**Issue by bebraw** [kenoxa/beamwind#3]
Based on the readme, you should use `import { getStyleTag } from '@beamwind/ssr'` to implement SSR.

The package doesn't seem to exist, though, and that's the only reference to `getStyleTag` within the repository if I understand correctly.

**Comment by sastan** [comment]
You are right. @beamwind/ssr does not exist at the moment. Below is a outline how it should/will be implemented:

```js
function getStyleTag(injector) {
  return `<style id="__beamwind">${injector.target.join('\n')}</style>`
}
```

The idea is that @beamwind/ssr would optimize the styles instead of simply joining them (see 455ccf1).

I hope that helps. Let me know if you need more details.

**Comment by bebraw** [comment]
@sastan Awesome, thanks. That makes sense. I'll go along this way then. Feel free to close this issue once you have the helper in the package. 👍

**Comment by bebraw** [comment]
I see you fixed this in f400cb3 so let's close.
